lib/vssh/curl_path.c

```diff
@@ -84,6 +84,12 @@ CURLcode Curl_getworkingpath(struct Curl_easy *data,
         return CURLE_OUT_OF_MEMORY;
       }
     }
+    else {
```

Comment:
I don't think this check needs to be more strict (e.g. `else if (working_path_len == 2 && !strcmp("/~", working_path))` because it's already restricted in the outer `if`, do you agree?

Like the only option for `working_path_len <= 2` here is `/~`